docs/articles/faq/README.md

```diff
@@ -259,10 +259,22 @@ This error can occur in the following situations:
 
 Check if the Web and DNS servers are online and configured to accept requests to this URL.
 
+Before it throws an exception, TestCafe waits for *two minutes* for the webpage to be served. Use one of the following options to increase this timeout:
+
+* [`--page-request-timeout` CLI option](../documentation/reference/command-line-interface.md#--cache)
+* [`pageRequestTimeout` configuration file option](../documentation/reference/configuration-file.md#pagerequesttimeout)
+* [`pageRequestTimeout` option of the `runner.run` method](../documentation/reference/testcafe-api/runner/run.md)
+
+TestCafe can retry failed network request for webpages. Use one of the following options to enable retries:
+
+* [`--retry-test-pages` CLI option](../documentation/reference/command-line-interface.md#--retry-test-pages)
+* [`retryTestPages` configuration file option](../documentation/reference/configuration-file.md#retrytestPages)
+* [`retryTestPages` option of the `runner.run` method](../documentation/reference/testcafe-api/runner/run.md)
+
 #### Unstable or improperly configured network connection
 
 * Check the network connection's settings.
-* Ensure that your network equipment works properly. If possible, establish a direct connection to the Internet/Web server.
+* Check your network equipment. If possible, establish a direct connection to the Internet/Web server.
```

Comment:
what is a direct connection?
```suggestion
* Check your network equipment. If possible, establish a wired connection to the Internet/Web server.
```

Reply:
This is an odd recommendation. I'd rather:
```suggestion
* Check your network equipment and the connection to the Internet/Web server.
```

Reply:
Nice!